.circleci/config.yml

```diff
@@ -630,10 +630,10 @@ workflows:
           context: org-global
           requires:
             - setup
-            - test-unit
-            - test-coverage
-            - test-integration
-            - test-functional
+#            - test-unit
```

Comment:
Is this a temporary measure?

Reply:
Correct. I have only disabled tests on snapshots.

PRs and tagged releases will still have to pass tests.